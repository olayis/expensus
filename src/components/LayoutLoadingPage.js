import React from 'react';
import Layout from '../components/Layout';
import LoadingPage from '../components/LoadingPage';

const LayoutLoadingPage = () => (
  <div className='layout-container'>
    <Layout />
    <main className='main-content'>
      <LoadingPage />
    </main>
  </div>
);

export default LayoutLoadingPage;
