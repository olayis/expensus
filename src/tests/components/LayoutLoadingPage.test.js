import React from 'react';
import { shallow } from 'enzyme';
import LayoutLoadingPage from '../../components/LayoutLoadingPage';

test('should render LayoutLoadingPage correctly', () => {
  const wrapper = shallow(<LayoutLoadingPage />);
  expect(wrapper).toMatchSnapshot();
});
