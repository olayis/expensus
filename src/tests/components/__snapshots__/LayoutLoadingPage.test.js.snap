// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LayoutLoadingPage correctly 1`] = `
<div
  className="layout-container"
>
  <Layout />
  <main
    className="main-content"
  >
    <LoadingPage />
  </main>
</div>
`;
